code/modules/cargo/packs/emergency.dm

```diff
@@ -130,7 +130,7 @@
 
 /datum/supply_pack/emergency/bomb
 	name = "Explosive Emergency Crate"
-	desc = "Science gone bonkers? Beeping behind the airlock? Buy now and be the hero the station des... I mean needs! (Time not included.)"
+	desc = "Science gone bonkers? Beeping behind the airlock? Buy now and become the hero the station ki-... I mean needs! Time not included, but a full bomb suit and hood, as well as a mask and defusal kit are! Non-Nuclear ordinance only."
```

Comment:
```suggestion
	desc = "Science gone bonkers? Beeping behind the airlock? Buy now and be the hero the station des... I mean needs! (Time not included. Not meant for nuclear ordnances.)"
```
Ruining a funny description are we?

Reply:
The whole point of the change was to tell people what is in the box.